environment-api/src/test/java/com/sequenceiq/environment/api/v1/environment/model/request/aws/AwsDiskEncryptionParametersTest.java

```diff
@@ -0,0 +1,84 @@
+package com.sequenceiq.environment.api.v1.environment.model.request.aws;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+class AwsDiskEncryptionParametersTest {
+
+    private static final String ENCRYPTION_KEY_ARN = "encryptionKeyArn";
+
+    @Test
+    void defaultConstructorTest() {
+        AwsDiskEncryptionParameters underTest = new AwsDiskEncryptionParameters();
+        underTest.setEncryptionKeyArn(ENCRYPTION_KEY_ARN);
+        verifyFields(underTest);
+    }
+
+    private void verifyFields(AwsDiskEncryptionParameters underTest) {
+        assertThat(underTest.getEncryptionKeyArn()).isEqualTo(ENCRYPTION_KEY_ARN);
+    }
+
+    @Test
+    void builderTest() {
+        AwsDiskEncryptionParameters underTest = AwsDiskEncryptionParameters.builder()
+                .withEncryptionKeyArn(ENCRYPTION_KEY_ARN)
+                .build();
+        verifyFields(underTest);
+    }
+
+    static Object[][] encryptionKeyArnValidationTestDataProvider() {
+        return new Object[][]{
+                // testCaseName encryptionKeyArn expectedValid
+                {"encryptionKeyArn=arn:aws:kms:us-east-1:012345678910:key/", "arn:aws:kms:us-east-1:012345678910:key/", false},
```

Comment:
Your test case name does not say anything about what going to be tested. Like: missing resource, missing servicename, missing accountid, wrong accountid, wrong UUID in key, etcetc.
I do not see test for a wrong UUID for example (maybe it is there but hard to see)

Reply:
Added the meaningful test case names.